pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -118,6 +118,10 @@ func (l *ScrapeConfigList) DeepCopyObject() runtime.Object {
 // ScrapeConfigSpec is a specification of the desired configuration for a scrape configuration.
 // +k8s:openapi-gen=true
 type ScrapeConfigSpec struct {
+	// The value of the `job` label assigned to the scraped metrics by default.
+	// +optional
+	// +kubebuilder:validation:MinLength=1
+	JobName *string `json:"jobName,omitempty"`
```

Comment:
What do you think about mentioning that the job label is configured via relabelings? I think people might get different expectations if this is not documented somewhere 🤔

Reply:
I agree, at the first glance one might think otherwise. Would be valuable to document it as well.

Reply:
I agree that additional details wouldn't hurt. It should also be noted that it won't change the `job_name` of the Prometheus scrape configuration which is always controlled by the operator to avoid duplicate job names which Prometheus doesn't support.